docs/app-store-connect/publish.md

```diff
@@ -49,6 +50,10 @@ The locale code name for displaying localized "What's new" content in TestFlight
 
 
 Describe the changes and additions to the build and indicate the features you would like your users to tests. If not given, the value will be checked from environment variable `APP_STORE_CONNECT_WHATS_NEW`. Alternatively to entering` WHATS_NEW `in plaintext, it may also be specified using a `@env:` prefix followed by a environment variable name, or `@file:` prefix followed by a path to the file containing the value. Example: `@env:<variable>` uses the value in the environment variable named `<variable>`, and `@file:<file_path>` uses the value from file at `<file_path>`.
+##### `--beta-build-localizations=BETA_BUILD_LOCALIZATIONS`
+
+
+Localized beta test info for what's new in the uploaded build as JSON encoded list. For example `[{"locale": "en-US", "whats_new": "What's new in english"}]`. See `--locale` for possible locale options. If not given, the value will be checked from environment variable `APP_STORE_CONNECT_BETA_BUILD_LOCALIZATIONS`. Alternatively to entering` BETA_BUILD_LOCALIZATIONS `in plaintext, it may also be specified using a `@env:` prefix followed by a environment variable name, or `@file:` prefix followed by a path to the file containing the value. Example: `@env:<variable>` uses the value in the environment variable named `<variable>`, and `@file:<file_path>` uses the value from file at `<file_path>`.
```

Comment:
```suggestion
Localized beta test info for what's new in the uploaded build as a JSON encoded list. For example, `[{"locale": "en-US", "whats_new": "What's new in english"}]`. See `--locale` for possible locale options. If not given, the value will be checked from the environment variable `APP_STORE_CONNECT_BETA_BUILD_LOCALIZATIONS`. Alternatively to entering ` BETA_BUILD_LOCALIZATIONS` in plaintext, it may also be specified using the `@env:` prefix followed by an environment variable name, or the `@file:` prefix followed by a path to the file containing the value. Example: `@env:<variable>` uses the value in the environment variable named `<variable>`, and `@file:<file_path>` uses the value from file at `<file_path>`.
```

Reply:
Partially resolved in 074cc8c. For the other part there is an open [issue](https://github.com/codemagic-ci-cd/cli-tools/issues/127).